eml/services/utilityFunctions.js

```diff
@@ -241,13 +241,7 @@ export function findIndexOfUncompletedComp(student, courseId, sectionId) {
     return -1; // or any other appropriate value to indicate no components
   }
 
-  const indexOfUncompletedComp = section.components.findIndex(component => !component.isComplete);
-
-  if (indexOfUncompletedComp === -1) {
-    console.log(`All components in section ${sectionId} of course ${courseId} are complete.`);
-  }
-
-  return indexOfUncompletedComp;
+  return section.components.findIndex(component => !component.isComplete);
 }
```

Comment:
just to make sure that this should be changed.. have you checked that it works?

Reply:
We removed the console.log (line 246-248) and then it is again assigned to a variable before returning the variable. Now it just returns it, no need for an extra step with no purpose.